schemas/get-aws-state-response.yml

```diff
@@ -0,0 +1,6 @@
+id:       http://schemas.taskcluster.net/aws-provisioner/v1/get-aws-state-response.json#
+$schema:  http://json-schema.org/draft-04/schema#
+title:                      "Get AWS State Response"
+description: |
+  All of the launch specifications for a worker type
+type:                       object
```

Comment:
Added to bug, that would be totally cool to use: https://nodejs.org/api/util.html#util_util_deprecate_function_string